src/sourmash/sourmash_args.py

```diff
@@ -22,11 +22,44 @@
 from . import signature as sigmod
 from .picklist import SignaturePicklist, PickStyle
 from .manifest import CollectionManifest
+import argparse
```

Comment:
is argparse needed or used below?

Reply:
Yes, I've used it in these functions: `check_scaled_bounds` and `check_num_bounds` to do this: `raise argparse.ArgumentTypeError`

Reply:
oh, ok, I missed that! 👍